docs/config/01-overview.md

```diff
@@ -123,7 +123,7 @@ export default function() { // or export default async function()
 ```
 
 :::note
-The exported function can also be `async`.
+The exported function can also be `async` but be aware that this can have implications on the performance of your application as all rendering operations are being executed after the global script finishes.
```

Comment:
```suggestion
The exported function can also be `async` but be aware that this can have implications on the performance of your application as all rendering operations will be deferred until after the global script finishes.
```

just one possible wording suggestion here